src/dbos-executor.ts

```diff
@@ -58,6 +58,11 @@ export interface DBOSConfig {
   readonly application?: object;
   readonly dbClientMetadata?: any;
   readonly debugProxy?: string;
+  readonly http?: {
+    readonly cors_middleware?: boolean;
+    readonly credentials?: boolean;
+    readonly allowed_origins?: string[];
+  };
```

Comment:
I  think this should be part of the runtime config and passed all the way down to the httpServer

Reply:
What do you mean by that?